src/main/java/games/strategy/thread/ThreadPool.java

```diff
@@ -19,9 +20,6 @@
    * fewer threads at any given time.
    */
   public ThreadPool(final int max) {
-    if (max < 1) {
-      throw new IllegalArgumentException("Max must be >= 1, instead it's:" + max);
-    }
     executorService = Executors.newFixedThreadPool(max);
```

Comment:
Executors.newFixedThreadPool already throws an IllegalArgumentException for max <= 0

Reply:
Only because this constructor is public, it might be helpful to document the precondition.

And since you're now relying on the underlying implementation to validate the precondition, a one-line unit test would help to ensure the precondition isn't accidentally removed in a future refactoring (there's already a `ThreadPoolTest` class).